### PR TITLE
Add filename parameter support to PDF endpoint

### PR DESCRIPTION
**Pull request by ksdme**
This adds the ability to set the receiving filename as part of the request itself. This patch does not update the README instructions but I am open to fixing it if this PR gets a greenlight.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -33,19 +33,24 @@ app.use(async (req, res, next) => {
     switch (type) {
       case 'pdf':
         const urlObj = new URL(url)
-        let filename = urlObj.hostname
-        if (urlObj.pathname !== '/') {
-          filename = urlObj.pathname.split('/').pop()
-          if (filename === '') filename = urlObj.pathname.replace(/\//g, '')
-          const extDotPosition = filename.lastIndexOf('.')
-          if (extDotPosition > 0) filename = filename.substring(0, extDotPosition)
+        if (!filename || String(filename).length === 0) {
```
Comment (zenato) [inline src/index.js]: This code could be replaced with ```if (!filename) {```

Comment (ksdme) [reply src/index.js]: `!''` evalues to `true`. `.pdf` wouldn't be a good name, right?

Comment (zenato) [reply src/index.js]: If `filename` parameter receive empty string, filename would be `urlObj.hostname`.pdf.

Comment (ksdme) [reply src/index.js]: Sorry, you are right.

Comment (ksdme) [reply src/index.js]: Updated.

**Pull request merged**